modules/phase_field/doc/content/modules/phase_field/Anisotropy.md

```diff
@@ -0,0 +1,17 @@
+# Anisotropy
```

Comment:
As this isn't complete, can you put a filler in here?

Reply:
Would it make sense to just list the syntax for these objects? What's that extension called?

Reply:
nvm, added some links and basic descriptions